pages/partners/index.tsx

```diff
@@ -96,6 +114,13 @@ const Partners: React.FC<{}> = () => {
           ))}
         </Flex>
       </div>
+
+      <div data-cy="partners-footer">
+        <Text mt="32px" textAlign="center">
```

Comment:
not the biggest fan of using px for css, would prefer em, rm or some other constant/multiple

Reply:
not the biggest fan of u

also we had this discussion a while ago i'm p sure and Varun mentioned this:
https://medium.com/hackernoon/rems-and-ems-and-why-you-probably-dont-need-them-664b9ce1e09f